platform/android/export/export.cpp

```diff
@@ -2016,6 +2018,13 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			ret = unzGoToNextFile(pkg);
 		}
 
+		if (!invalid_abis.empty()) {
+			String unsupported_arch = String(", ").join(invalid_abis);
+			EditorNode::add_io_error("lib for the selected architectures " + unsupported_arch + " is not found in the export template.\
+				\nPlease uncheck the unsupported architectures in the export settings or add the required libs in the export template.");
+			return ERR_FILE_NOT_FOUND;
```

Comment:
This should use `CLEANUP_AND_RETURN(ERR_FILE_NOT_FOUND);`, otherwise you'll leave a stray temporary APK in the cache.